cmd/snap/cmd_known.go

```diff
@@ -52,7 +55,12 @@ shown must also have the specified headers matching the provided values.
 func init() {
 	addCommand("known", shortKnownHelp, longKnownHelp, func() flags.Commander {
 		return &cmdKnown{}
-	}, nil, []argDesc{
+	}, map[string]string{
+		// TRANSLATORS: This should not start with a lowercase letter.
+		"remote": i18n.G("Query the store for the assertion via snapd"),
```

Comment:
Wouldn't this be more correct and clear if it said
```suggestion
		"remote": i18n.G("Query the store for the assertion, via snapd if possible"),
```

... (cont'd)